netebpfext/net_ebpf_ext_program_info.h

```diff
@@ -21,24 +21,33 @@ static ebpf_context_descriptor_t _ebpf_xdp_context_descriptor = {
     EBPF_OFFSET_OF(xdp_md_t, data),
     EBPF_OFFSET_OF(xdp_md_t, data_end),
     EBPF_OFFSET_OF(xdp_md_t, data_meta)};
+
 static ebpf_program_info_t _ebpf_xdp_program_info = {
     {"xdp", &_ebpf_xdp_context_descriptor, {0}, BPF_PROG_TYPE_XDP},
     EBPF_COUNT_OF(_xdp_ebpf_extension_helper_function_prototype),
     _xdp_ebpf_extension_helper_function_prototype};
+
 static ebpf_program_section_info_t _ebpf_xdp_section_info[] = {
     {(const wchar_t*)L"xdp", &EBPF_PROGRAM_TYPE_XDP, &EBPF_ATTACH_TYPE_XDP, BPF_PROG_TYPE_XDP, BPF_XDP}};
 
 // Bind program information.
 static ebpf_context_descriptor_t _ebpf_bind_context_descriptor = {
     sizeof(bind_md_t), EBPF_OFFSET_OF(bind_md_t, app_id_start), EBPF_OFFSET_OF(bind_md_t, app_id_end), -1};
+
 static ebpf_program_info_t _ebpf_bind_program_info = {
     {"bind", &_ebpf_bind_context_descriptor, {0}, BPF_PROG_TYPE_BIND}, 0, NULL};
+
 static ebpf_program_section_info_t _ebpf_bind_section_info[] = {
     {L"bind", &EBPF_PROGRAM_TYPE_BIND, &EBPF_ATTACH_TYPE_BIND, BPF_PROG_TYPE_BIND, BPF_ATTACH_TYPE_BIND}};
 
 // CGROUP_SOCK_ADDR global helper function prototypes.
```

Comment:
nit: This comment is for `_ebpf_sock_addr_global_helper_function_prototype` in line 50. We can add a new similar comment for the `program type specific` helper functions.

Reply:
fixed